### PR TITLE
Hints and notes added to handle grafana dashboards

### PR DESCRIPTION
**Pull request by eedugon**
Hello,

Here you have a small contribution with these changes:

- Point the directory where make generate should be executed from
- Mention that the saved dashboard file should be suffixed with "-dashboard.json".
- Added hints to apply the new configMap and restart grafana

In main README.md I have also added a link to developing-alerts-and-dashboards.md doc for more information.

Regards,
  Eduardo

### CODE REVIEW

File: contrib/kube-prometheus/docs/developing-alerts-and-dashboards.md [hunk 1]
```diff
@@ -22,6 +24,16 @@ The `ConfigMap` that is generated and holds the dashboard definitions can be fou
...
+Note: The dashboard json file to be copied in `assets/grafana/` should be suffixed with `-dashboard.json`, otherwise it won't be processed by `make generate`.
+
+Then the generated manifest can be applied against a Kubernetes cluster with something like:
+```
+kubectl -n monitoring apply -f manifests/grafana/grafana-dashboards.yaml
+```
+
+And to apply the changes, restart grafana:
+```
+kubectl -n monitoring delete pod -l app=grafana
```
Comment (brancz) [inline contrib/kube-prometheus/docs/developing-alerts-and-dashboards.md]: This should not be necessary. The grafana-watcher should pick up the changes of the dashboards and automatically reload them.

Comment (eedugon) [inline contrib/kube-prometheus/docs/developing-alerts-and-dashboards.md]: Absolutely, then I will remove that part. And i will also check if the behavior is correct.

Comment (eedugon) [inline contrib/kube-prometheus/docs/developing-alerts-and-dashboards.md]: confirmed it works as you described.

**Comment by brancz** [comment]
Awesome! Thanks a lot for your effort!

**Pull request merged**